src/main/java/io/jenkins/plugins/orka/IdleTimeCloudRetentionStrategy.java

```diff
@@ -29,14 +34,36 @@ public DescriptorImpl getDescriptor() {
     @Restricted(NoExternalUse.class)
     public static final DescriptorImpl DESCRIPTOR = new DescriptorImpl();
 
+    @Extension
     public static final class DescriptorImpl extends Descriptor<RetentionStrategy<?>> {
         @Override
         public String getDisplayName() {
             return "Keep until idle time expires";
+        } 
+        
+        public FormValidation doCheckIdleMinutes(@QueryParameter String value) {
+            try {
```

Comment:
We can write tests for this method here.

Reply:
Tests added. Will add more tests for other validation checks as well.